public/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
```

Comment:
Here in `content` you can write the text that you want to show in site description.
This will be visible when you will share the URL with others. Like this, 
![image](https://user-images.githubusercontent.com/56081584/161434946-5e264344-7f8d-4d05-8092-7b9f2666dbbd.png)